STX.REST.RESTFulSense.Clients/Services/Foundations/HttpExchanges/HttpExchangeService.Mappings.cs

```diff
@@ -0,0 +1,149 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using STX.REST.RESTFulSense.Clients.Models.Services.HttpExchanges;
+
+namespace STX.REST.RESTFulSense.Clients.Services.Foundations.HttpExchanges
+{
+    internal partial class HttpExchangeService
+    {
+        private static TOutput[] MapArray<TInput, TOutput>(
+            IEnumerable<TInput> elements,
+            Func<TInput, TOutput> functionMapping)
+        {
+            TOutput[] outputElements = null;
+            if (elements != null)
```

Comment:
```suggestion
            if (elements is not null)
```
Do the same for the rest in this file.

Reply:
Fixed

Reply:
Fixed